bundles/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/handler/RFXComHandler.java

```diff
@@ -145,13 +145,20 @@ public void onDeviceMessageReceived(ThingUID bridge, RFXComDeviceMessage message
                         String channelId = uid.getId();
 
                         try {
-                            if (channelId.equals(CHANNEL_LOW_BATTERY)) {
-                                updateState(uid, isLowBattery(message.convertToState(CHANNEL_BATTERY_LEVEL)));
-                            } else {
-                                State state = message.convertToState(channelId);
-                                if (state != null) {
```

Comment:
It looks you removed this part of the logic?

Reply:
No. There's a low battery case and a default that does a generic convert-and-update?

Reply:
I think @martinvw is looking for the null-check (old l. 152).